cluster/failover_config.go

```diff
@@ -92,12 +95,19 @@ func (c *FailoverConfig) Validate(rootConfig Config) error {
 	if len(c.Configs) == 0 {
 		return fmt.Errorf("at least one failover cluster config expected: %w", hzerrors.ErrIllegalArgument)
 	}
-	for i, cc := range c.Configs {
-		if err := cc.Validate(); err != nil {
+	for i := range c.Configs {
+		ccp := &c.Configs[i]
+		// if a cluster connection timeout was not specified, set it to default.
+		t := ccp.ConnectionStrategy.Timeout
+		if t == 0 {
```

Comment:
Instead of 0, it would make sense to compare against a well named static to make it more obvious why we are comparing against it. 
see https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java#L299

Reply:
Since the zero value of `ccp.ConnectionStrategy.Timeout` is `0` it is idiomatic to use it.

Reply:
0 is ok but it should have a name to make it more obvious to the readers like me. 
I would like to see it assign it to a constant and named something like DEFAULT_CONNECTION_TIMEOUT/UNSET_CONNECTION_TIMEOUT.